src/elexmodel/models/BaseElectionModel.py

```diff
@@ -264,3 +266,36 @@ def get_coefficients(self):
         These coefficients are for the point prediciton only, not for the lower or upper intervals models.
         """
         return self.features_to_coefficients
+
+    def compute_lambda(
```

Comment:
this is truly the tiniest thing, but do you mind changing the name of this function to `find_optimal_lambda`